src/engine/sidechain/shoutconnection.h

```diff
@@ -103,15 +102,15 @@ class ShoutConnection
 
     bool writeSingle(const unsigned char *data, size_t len);
 
-    QByteArray encodeString(const QString& string);
+    const QByteArray encodeString(const QString& string) const;
```

Comment:
const return values inhibit R-value optimizations (you can't move a const value in C++). Const (non-reference) return values seldomly make sense in C++11 and newer.
```suggestion
    QByteArray encodeString(const QString& string) const;
```

Reply:
Okay, thank you.